R/inbo_rapport.R

```diff
@@ -43,9 +49,9 @@ inbo_rapport <- function(
   floatbarrier = c(NA, "section", "subsection", "subsubsection"),
   codesize = c("footnotesize", "scriptsize", "tiny", "small", "normalsize"),
   style = c("INBO", "Vlaanderen", "Flanders"),
+  babel = "french,dutch,english",
```

Comment:
Somehow the syntax of the `babel` argument breaks the user's habit of providing vectors (or lists) to arguments with multiple values. Having a input vector here would make the function arguments more uniform; it could be concatenated by the code.

Reply:
I'll convert it to a vector.

Reply:
I've reworked this in 8e3c892